h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -42,6 +45,9 @@
   String[][] _na_strings;       // Strings for NA in a given column
   String[][] _data;           // First few rows of parsed/tokenized data
 
+  int tentativeNumLines = -1;
```

Comment:
This one is for investigating a stupid failure when line length is calculated to be 1.3M. Will get back to that case later on. Currently the line is commented out.